juristische-zitierweise.csl

```diff
@@ -40,7 +40,7 @@
     </names>
   </macro>
   <macro name="locator-with-label">
-    <group delimiter=" ">
+    <group delimiter="&amp;#160;">
```

Comment:
I added them by hand in the CSL editor and they still look fine within the editor:

![csl-editor-nonbreaking-space](https://user-images.githubusercontent.com/5199995/43779757-652087e2-9a59-11e8-893a-5aa28f4a4b54.png)
It might simply be that exporting the style from there is then escaping it wrongly.

Reply:
Okay, I checked and it is also in the Code editor wrongly escaped. I will fix these issues w/o the editor.

Reply:
Maybe copy and pasting the non-breaking space character into the delimiter field works. But yeah, it looks like the code escapes the ampersand sign.